Source/SocketExtensions.swift

```diff
@@ -31,7 +31,11 @@ enum JSONError : Error {
 
 extension Array {
     func toJSON() throws -> Data {
-        return try JSONSerialization.data(withJSONObject: self, options: JSONSerialization.WritingOptions(rawValue: 0))
+        if #available(iOSApplicationExtension 11.0, *) {
```

Comment:
I think you might need more checks here. Does this compile for tvOS/macOS targets?

Reply:
You are correct, added the fix